src/index.js

```diff
@@ -1,21 +1,40 @@
-import _ from 'lodash';
-import printMe from './print.js';
+// import _ from 'lodash';
+// import printMe from './print.js';
+import Library from './modules/libraryModule.js';
 import './style.scss';
 
-function component() {
-  const element = document.createElement('div');
-  const btn = document.createElement('button');
+// const libraryBtn = document.getElementById('LibraryBtn');
+// const newBtn = document.getElementById('NewBtn');
+// const libraryPage = document.getElementById('Library');
+// const newPage = document.getElementById('New');
+// const libraryStr = 'library';
+// const hidden = 'hidden';
+// const newStr = 'new';
+// const contactStr = 'contact';
+// const pageSetup = () => {
+//   // newPage.className = hidden;
+//   // contactPage.className = hidden;
+// };
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['JT\'s', 'To-do List'], ' ');
-  element.classList.add('hello');
+// pageSetup();
 
-  btn.innerHTML = 'Click me and check the console!';
-  btn.onclick = printMe;
+const library = new Library();
+library.setUpAddListener();
 
-  element.appendChild(btn);
+// function component() {
+//   const element = document.createElement('div');
+//   const btn = document.createElement('button');
 
-  return element;
-}
+//   // Lodash, now imported by this script
+//   element.innerHTML = _.join(['JT\'s', 'To-do List'], ' ');
 
-document.body.appendChild(component());
+//   btn.innerHTML = 'Click me and check console';
+//   btn.onclick = printMe;
+//   btn.classList.add('clear-all-btn');
+//   element.classList.add('hello');
+//   element.appendChild(btn);
+
+//   return element;
+// }
+
+// document.body.appendChild(component());
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable.

Reply:
> * [ ]  Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable.

Done, thank you :-)